detekt-rules-style/src/main/kotlin/io/gitlab/arturbosch/detekt/rules/style/ForbiddenSuppress.kt

```diff
@@ -0,0 +1,91 @@
+package io.gitlab.arturbosch.detekt.rules.style
+
+import io.gitlab.arturbosch.detekt.api.CodeSmell
+import io.gitlab.arturbosch.detekt.api.Config
+import io.gitlab.arturbosch.detekt.api.Debt
+import io.gitlab.arturbosch.detekt.api.Entity
+import io.gitlab.arturbosch.detekt.api.Issue
+import io.gitlab.arturbosch.detekt.api.Rule
+import io.gitlab.arturbosch.detekt.api.Severity
+import io.gitlab.arturbosch.detekt.api.config
+import io.gitlab.arturbosch.detekt.api.internal.Configuration
+import org.jetbrains.kotlin.psi.KtAnnotationEntry
+import org.jetbrains.kotlin.psi.KtLiteralStringTemplateEntry
+import org.jetbrains.kotlin.psi.KtStringTemplateExpression
+import org.jetbrains.kotlin.psi.KtValueArgument
+import org.jetbrains.kotlin.psi.KtValueArgumentList
+
+/**
+ * This rule allows to set a list of rules whose suppression is forbidden. This can be used to discourage abuse of the
+ * `Suppress` and `SuppressWarnings` annotations. Detekt will report suppression of all forbidden rules. This rule is
+ * not capable of reporting suppression of itself, as that's a language feature with precedence.
```

Comment:
minor formatting and grammar

```suggestion
 * This rule allows to set a list of rules whose suppression is forbidden.
 * This can be used to discourage the abuse of the `Suppress` and `SuppressWarnings` annotations.
 * Detekt will report the suppression of all forbidden rules.
 * This rule is not capable of reporting the suppression of itself, as that's a language feature with precedence.
```